README.md

```diff
@@ -161,6 +161,17 @@ Prometheus.
 const promQL = new PromQLExtension().setComplete({ remote: { httpErrorHandler: (error: any) => console.error(error) } })
 ```
 
+###### HTTP method used
+
+By default, the prometheus client will use the HTTP method `POST` when contacting Prometheus for the
+endpoints `/api/v1/labels` and `/api/v1/series`.
+
+You can change it to use the HTTP method `GET` if you prefer.
+
+```typescript
+const promQL = new PromQLExtension().setComplete({ remote: { httpMethodUsed: 'get' } })
```

Comment:
I feel `httpMethodUsed` is unnecessarily verbose. Better IMO would be just `method` or `httpMethod`.